cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/common/ExtensionBuilders.java

```diff
@@ -108,50 +109,49 @@ public static IBaseBooleanDatatype buildBooleanType(FhirVersionEnum fhirVersion,
 
     @SuppressWarnings("unchecked")
     public static <T extends IBaseExtension<?, ?>> T buildSdcLaunchContextExt(
-            FhirVersionEnum fhirVersion, String code, String resourceType) {
+            FhirVersionEnum fhirVersion, String code, List<String> resourceTypes) {
         var system = "http://hl7.org/fhir/uv/sdc/CodeSystem/launchContext";
         var display = "";
         switch (code) {
             case "patient":
                 display = "Patient";
-                resourceType = display;
+                resourceTypes = Arrays.asList(display);
                 break;
             case "encounter":
                 display = "Encounter";
-                resourceType = display;
+                resourceTypes = Arrays.asList(display);
```

Comment:
I strongly suggest you clone resourceTypes into a new variable, like resourceTypesToUse, instead of hijacking the constructor argument.